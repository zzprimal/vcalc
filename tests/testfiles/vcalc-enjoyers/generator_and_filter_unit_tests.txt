vector a = [i in 0..20 | i];
vector i = [i in 0..6 | i];
print(a + i);
print([a in a | a*2 + 1]);
print([a in i | a*2 + 1]);
print([i in a | i*2 + 1]);
print([i in [i in [i in [i in a+i[4]..17 | i] | i] | i] | i*2 + 1]);
print([i in [i in [i in [i in 12..3 | i] | i] | i] | i*2 + 1]);
print([a in a | a*2 + 1] + [i in 0..20 | i]);
print([a in a | a*2 + 1] - [i in 0..20 | i]);
print([a in a | a*2 + 1] * [i in 0..20 | i]);
print([a in a | a*2 + 1] / [i in 1..20 | i]);
print([a in a | a*2 + 1] == [i in 0..20 | i]);
print([a in a | a*2 + 1] != [i in 0..20 | i]);
print([a in a | a*2 + 1] > [i in 0..20 | i]);
print([a in a | a*2 + 1] < [i in 0..20 | i]);
vector b = 32436..32436;
if (1)
    b = [Y in 1..6 & Y];
fi;
int Y = 2;
print(Y);
Y = 3;
//CHECK_FILE:./generator_and_filter_unit_tests.out
