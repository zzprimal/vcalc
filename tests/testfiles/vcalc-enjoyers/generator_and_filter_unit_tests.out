[0 2 4 6 8 10 12 7 8 9 10 11 12 13 14 15 16 17 18 19 20]
[1 3 5 7 9 11 13 15 17 19 21 23 25 27 29 31 33 35 37 39 41]
[1 3 5 7 9 11 13]
[1 3 5 7 9 11 13 15 17 19 21 23 25 27 29 31 33 35 37 39 41]
[9 13 17 21 25 29 33 37 41 45 49 53 57 61 29 31 33 35 37 39 41]
[]
[1 4 7 10 13 16 19 22 25 28 31 34 37 40 43 46 49 52 55 58 61]
[1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21]
[0 3 10 21 36 55 78 105 136 171 210 253 300 351 406 465 528 595 666 741 820]
[1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 41]
[0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0]
[1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1]
[1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1]
[0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0]
2
